deployments/kubernetes/install/helm/istio-cni/templates/istio-cni.yaml

```diff
@@ -164,3 +191,33 @@ subjects:
   name: istio-cni
   namespace: {{ .Release.Namespace }}
 {{- end }}
+    {{- if .Values.repair.enabled }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: istio-cni-repair-role
+  namespace: {{ .Release.Namespace}}
+  labels:
+    k8s-app: istio-cni-repair
+rules:
+- apiGroups: [""]
+  resources: ["pods", "events"]
+  verbs: ["get", "list", "watch", "delete", "patch", "update", "create" ]
```

Comment:
Does this daemonset really need the permission to create pods in any namespace?
Trim this list to the minimum.
You should probably separate the `"pods"` and `"events"` permissions to have different verbs for each.

Reply:
Split.